src/dfx/src/commands/canister/create.rs

```diff
@@ -196,10 +207,12 @@ pub async fn exec(
                 .with_context(|| {
                     format!("Failed to read freezing threshold of {}.", canister_name)
                 })?;
+                // TODO: create for pull canisters with their mainnet ID
```

Comment:
Is there a JIRA ticket that tracks this TODO? If so, please link to it in this comment. If not, please create one.